site/src/app/VSCodeButton.tsx

```diff
@@ -20,7 +21,7 @@ function VSCodeButton({ app }: Props): JSX.Element {
 
     return (
         <a
-            href={`vscode://nordic-semiconductor.nrf-connect-extension-pack/get-application?${params.toString()}`}
+            href={`vscode://nordic-semiconductor.nrf-connect-extension-pack/execute-command?${params.toString()}`}
```

Comment:
Why not just keep it as is, but make the URI handler execute them all as commands with the `nrf-connect` prefix?

Reply:
I prefer the explicitness of this approach really.